docs/limitations/nested-objects.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[nested-objects]]
+== Limitation with nested objects
+
+Kibana is not able to perform aggregations across fields containing nested
```

Comment:
I'd tweak this slightly to make it easier to parse: "Kibana cannot perform aggregations across fields that contain nested objects."